### PR TITLE
Decrease h1 font-size and make footer center-bottom

### PR DESCRIPTION
**Pull request by wildan3105**

### CODE REVIEW

**Comment by wildan3105** [comment]
Edited it

File: assets/css/immersion.scss [hunk 1]
```diff
@@ -64,8 +64,14 @@ a:hover,
...
 
+.site-footer {
+  font-size:14px;
+  text-align:center;
+  padding-bottom:10px;
+}
+
 .project-name {
-  font-size: 6em;
+  font-size:60px;
```
Comment (mkasberg) [inline assets/css/immersion.scss]: Again `em` or `%` for font size.

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,4 @@ _site
 # Ruby Gems & Bundler Stuff for Jekyll
 .bundle
 vendor
+Gemfile.lock
```
Comment (mkasberg) [inline .gitignore]: `Gemfile.lock` is supposed to be checked in. Leave it out of the `.gitignore`.

http://bundler.io/v1.3/rationale.html#checking-your-code-into-version-control

File: assets/css/immersion.scss [hunk 3]
```diff
@@ -64,8 +64,14 @@ a:hover,
   margin-bottom: 40px;
 }
 
+.site-footer {
+  font-size:14px;
```
Comment (mkasberg) [inline assets/css/immersion.scss]: We should use `em`s or `%`s for font sizes here.

https://kyleschaeffer.com/development/css-font-size-em-vs-px-vs-pt-vs/

**Comment by wildan3105** [comment]
Sorry for 2nd mistake. Edited it.

**Comment by mkasberg** [comment]
Thanks for your work!

**Comment by wildan3105** [comment]
Thanks for the merge ! :+1:

**Pull request merged**